templates/admin/release_report_detail.html

```diff
@@ -57,7 +57,7 @@ <h1>Boost</h1>
         </div>
         <div class="flex flex-col h-full gap-y-4">
 
-          {% if version.release_report_cover_image.url %}
+          {% if version.release_report_cover_image %}
```

Comment:
Safer? Overkill?

```suggestion
          {% if version.release_report_cover_image and version.release_report_cover_image.url %}
```

Reply:
Fine with me

Reply:
I don't think it's possible to have a non-null `FileField` instance and have it return a falsey `url`, so probably overkill.